PIL/Image.py

```diff
@@ -1092,12 +1154,14 @@ def draft(self, mode, size):
         pass
```

Comment:
Well, while it's possible to infer length while iterating, these are supposed to be length==2 so it makes sense to assume it might be tested (which it currently is not?)

@wiredfool Re django etc, does this mean that all such `Size` usage should not be a tuple?